setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+# Allow for longer test strings. Code is formatted to 88 columns by Black.
+max-line-length = 99
```

Comment:
I'm still in the boat of using the program's default config file.  Even when using the 'single config file for everything' option you still end up with multiple single config files.

Reply:
The plan it to move more things from setup.py into setup.cfg ... I am using it in #398

Reply:
Any project using black will still have to have a `pyproject.toml` for any config there, afaik.  Anyways, not a bug, just a passing opinion.